src/VecSim/algorithms/brute_force/bfm_batch_iterator.h

```diff
@@ -6,7 +6,7 @@
 
 class BFM_BatchIterator : public BF_BatchIterator {
 public:
-    BFM_BatchIterator(void *query_vector, const BruteForceIndex *index,
+    BFM_BatchIterator(void *query_vector, const BruteForceIndex<float, float> *index,
```

Comment:
This causes some compile errors and is not really necessary because there should not be any difference between the classes besides implementation. I might change it after we add the batch iterator factory.

Reply:
K
what is the error? just out of curiosity